src/RoadRegistry.BackOffice/Messages/Polygon.cs

```diff
@@ -0,0 +1,8 @@
+namespace RoadRegistry.BackOffice.Messages
+{
+    public class Polygon
+    {
+        public Ring Ring { get; set; }
```

Comment:
Would you prefer the word `Shell` here? I'm fine with this as well, just not sure what "common" word is used for it.

Reply:
Shell seems more explicit, let's go for that.